examples/clusterautoscaler.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: "autoscaling.openshift.io/v1alpha1"
+kind: "ClusterAutoscaler"
+metadata:
+  name: "example"
+spec:
+  minReplicas: 1
```

Comment:
Ah - The min and max replicas here is for the autoscaler itself. Yes?

Reply:
Actually was expecting this to represent that min and max for the target MachineSet. Just commented in the other thread with my confusion about the plan here though.